tools/apiview/parsers/swagger-api-parser/SwaggerApiParser/SwaggerApiView/CodeFile.cs

```diff
@@ -0,0 +1,133 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text.Json;
+using System.Threading.Tasks;
+
+namespace SwaggerApiParser;
+
+public class CodeFile
+{
+    private static readonly JsonSerializerOptions JsonSerializerOptions = new JsonSerializerOptions() {AllowTrailingCommas = true, ReadCommentHandling = JsonCommentHandling.Skip};
+
+    private string _versionString;
+
+    private static HashSet<string> _collapsibleLanguages = new HashSet<string>(new string[] {"Swagger"});
+
+    [Obsolete("This is only for back compat, VersionString should be used")]
+    public int Version { get; set; }
```

Comment:
This property is not required if we are not using it.

Reply:
Removed version